src/posit/connect/bundles.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import annotations
+
+from typing import List
+
+from requests.sessions import Session as Session
+
+from posit.connect.config import Config
+
+from . import urls
+
+from .resources import Resources, Resource
+
+
+class BundleMetadata(Resource):
```

Comment:
Does this need to be a Class? It's not a Resource properly, is it? There's nothing you can do with it. It just feels like a nested field in `Bundle` to me, `Bundle.metadata` is a dict.